data-es6.js

```diff
@@ -3569,9 +3590,10 @@ exports.tests = [
       exec: function() {/*
         var obj = { foo: 1, bar: 2 };
         var iterator = Reflect.enumerate(obj);
-
+        var passed = 1
+        if(typeof Symbol != undefined && 'iterator' in Symbol)passed &= Symbol.iterator in iterator;
```

Comment:
Can you please make it clean and consistent with the rest of the code — semicolon after `passed = 1`, space after `if`, `passed &=` on new line and wrapped in block.